documentation/API-onboarding.md

```diff
@@ -0,0 +1,75 @@
+
+The Steering Committee documents the decision in the [CAMARA API overview list](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIBacklog.md) (fills in columns Steering Committee date and Steering Committee decision / Sub Project; in case of a No-Go "Rejected" is documented there).
+
+## API management and maintenance
+
+In case of a GO decision of the Steering Committee the named Sub Project takes over responsibility for the new API or API family and starts making contributions for the API definition(s), API documentation, provider implementation and test artefacts.
+
+The Sub Project will follow the versioning guidelines as defined in the Commonalities Working Group to maintain the API versions. Based on an internal consensus within the Sub Project, releases will be cherry-picked and release branches will be appropriately created. If a deliverable (API definition, API documentation, API code, successful test) for a release of an API is available (accepted by the Sub Project and merged in the main branch) the Sub Projects adds the link to it in the [CAMARA API overview list](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIBacklog.md).
+
+If there are more than one API in an API family / API Sub Project for each of them a separate line in the [CAMARA API overview list](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIBacklog.md) shall be added.
+
+If there are more than one releases for an API for each of these releases a separate line in the [CAMARA API overview list](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIBacklog.md) shall be added.
+
+The requested data to be filled in the [CAMARA API overview list](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIBacklog.md) is API name, API release, link to API definition, link to API documentation, link to API code and partner which has tested an API release.
+
+## API deployment
+
+If an API is successfully tested in one or more operator networks, it can be used for deployment also in live environments.
+
+In case an operator (CAMARA partner company or external company) uses an API definition of a CAMARA API release, this operator is recommended to document it in the last column (API deployed at company and country) of the [CAMARA API overview list](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIBacklog.md). If possible, also a link to the site (e.g. portal) where the deployed API can be accessed is recommended to be inserted.
+
+## Clean up
+
+Lines for old releases of APIs (when these aren't deployed anywhere anymore) in the [CAMARA API overview list](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIBacklog.md) can be deleted by the responsible API Sub Project.
```

Comment:
> tracking actual API deployment may be hard ... It might be easier to state CAMARA maintains API versions N, N-1, N&2, and archives older ones. The corresponding branches should be identified as archived.

Yes, that's the idea precisely captured in the live table proposed in [PR#131](https://github.com/camaraproject/WorkingGroups/pull/131/commits/4b71fcc944abed1c5b9e8ca3f325863a39d1df11)

Reply:
@tanjadegroot @jordonezlucena : I'll add a sentence that it is clear target only to have 3 releases  N, N-1, N&2, and to force API implementors to update by archiving older ones.

Reply:
Do we have "API publication readiness" criteria and an API publication & announcement process ? Could be using existing processes, but just documenting which one to apply to publish an API / new API version